examples/appengine/hello.go

```diff
@@ -2,10 +2,10 @@ package mellow
 
 import (
 	"fmt"
-	"github.com/bugsnag/bugsnag-go"
-	"github.com/bugsnag/bugsnag-go/errors"
 	"net/http"
 	"os"
+
+	"github.com/bugsnag/bugsnag-go"
```

Comment:
Oh neat, I definitely agree that separating stdlib from third-party libs is better, though I was under the impression that `gofmt` didn't have an option for this. How did you make it configurable? (0_ō)